hidimstat/clustered_inference.py

```diff
@@ -154,17 +156,21 @@ def clustered_inference(X_init, y, ward, n_clusters, train_size=0.3,
     beta_hat : ndarray, shape (n_features,) or (n_features, n_times)
         Estimated parameter vector or matrix.
 
-    sf : ndarray, shape (n_features,)
-        Survival function values of every feature.
+    pval : ndarray, shape (n_features,)
+        p-values, testing the negativity (low p-values means
```

Comment:
what do you mean by "negativity"? rejecting the null hypothesis?

Reply:
rejecting negativity means the effect is positive.

Reply:
In this context, "rejecting" stands for "rejecting the null hypothesis", hence you cannot says "rejecting negativity".

Reply:
In general you reject an Hypothesis like H0: param <= 0 vs H1: param > 0. That was for this reason I was using this terminology but I will make it more straight forward as requested ! ;)